ui/components/multichain/pages/send/send.test.js

```diff
@@ -35,10 +35,8 @@ const render = (props = {}) => {
 describe('SendPage', () => {
   describe('render', () => {
     it('renders correctly', () => {
-      const { container, getByTestId } = render();
+      const { container } = render();
       expect(container).toMatchSnapshot();
-
-      expect(getByTestId('send-page-network-picker')).toBeInTheDocument();
```

Comment:
Why are we removing this expectation?

Reply:
added this back and the tests are passing too. :(